statsmodels/tsa/automatic/sarimax.py

```diff
@@ -44,8 +44,12 @@ def auto_order(endog, criteria='aic', d=0, max_order=(3, 3),
         for p in range(max_order[0]):
             for q in range(max_order[1]):
                 # fit  the model
-                mod = sm.tsa.statespace.SARIMAX(endog, order=(p, d, q), **spec)
-                res = mod.fit(disp=False)
+                try:
+                    mod = sm.tsa.statespace.SARIMAX(endog,
+                                                    order=(p, d, q), **spec)
+                    res = mod.fit(disp=False)
+                except Exception as e:
+                    raise e
```

Comment:
Rather than re-raising this exception, it's probably better to issue a warning (`warnings.warn('Count not fit model with p={} and q={}')`) and then set `aic_matrix[p, q] = np.inf` so that it is definitely not selected by the function.

Reply:
Sure, I'll do that.